routes/index.js

```diff
@@ -46,4 +48,7 @@ module.exports = (app) => {
 
   app.route('/khairul')
     .get(khairulController.index);
+  
+  app.route('/ali')
+    .get(aliController.b);
```

Comment:
You have to call the function that you previously bind in your controller here. That function is `index`. This line should be `aliController.index`.